src/couch_quickjs/test/couch_quickjs_tests.erl

```diff
@@ -0,0 +1,71 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License. You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+-module(couch_quickjs_tests).
+
+-include_lib("couch/include/couch_eunit.hrl").
+
+setup() ->
+    Ctx = test_util:start_couch(),
+    Ctx.
+
+teardown(Ctx) ->
+    config:delete("quickjs", "memory_limit_bytes", _Persist = false),
+    test_util:stop_couch(Ctx).
+
+quickjs_test_() ->
+    {
+        foreach,
+        fun setup/0,
+        fun teardown/1,
+        case os:type() of
+            {win32, _} ->
+                [];
```

Comment:
Since compiling for Windows is working now, should we run the tests there too?

Reply:
good find!

Reply:
Updated and now windows tests are running.